vimeo/vimeo-networking-java
# VA-4206: publish to social objects

#### Issue
Implementation of the Publish to Social data objects from the [Publish to Social design](https://github.vimeows.com/Vimeo-API/design/pull/148)


#### Summary
Added The Publish To Social data objects to each version of the networking library. 

#### How to Test
CI branch: https://publish-endpoints-api.ci.vimeows.com/

For testing I found it easiest to add a break point to line 183 in `BaseWatchStreamAdapter` when looking at the `RecentVideosStreamFragment` and evaluate the video object  to confirm it contained all the expected values in the new models.
